docker-compose.yml

```diff
@@ -125,14 +125,10 @@ services:
       - ./tests/frontend:/opt/warehouse/src/tests/frontend:z
       - ./bin:/opt/warehouse/src/bin:z
 
-  smtp:
-    build:
-      context: .
-    command: python /opt/warehouse/dev/smtp.py 0.0.0.0:2525
+  maildev:
+    image: maildev/maildev
     ports:
-      - "2525:2525"
-    volumes:
-      - ./dev/smtp.py:/opt/warehouse/dev/smtp.py
+      - "1080:80"
```

Comment:
I haven't exposed the 25 port externally because the worker container doesn't need port forwarding to access it, but if you have the habit of exposing ports to the host, e.g. so that you can run the worker outside of docker, let me know :)